sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/AnsiCastSuiteBase.scala

```diff
@@ -618,10 +618,14 @@ class AnsiCastSuiteWithAnsiModeOn extends AnsiCastSuiteBase {
   }
```

Comment:
I think we should clean up the test suites a bit. Now we only need to test `Cast` and `TryCast`